massgen/tests/test_plan_storage.py

```
@@ -152,6 +152,26 @@ def test_get_latest_plan(self, temp_plans_dir):
         assert latest is not None
         assert latest.plan_id == session2.plan_id
 
+    def test_get_latest_resumable_plan(self, temp_plans_dir):
```

Ruff (0.15.0)

[warning] Unused method argument: `temp_plans_dir`

(ARG002)